skimage/util/_regular_grid.py

```diff
@@ -65,7 +65,8 @@ def regular_grid(ar_shape, n_points):
     space_size = float(np.prod(ar_shape))
     if space_size <= n_points:
         return (slice(None), ) * ndim
-    stepsizes = (space_size / n_points) ** (1.0 / ndim) * np.ones(ndim)
+    stepsizes = np.full(ndim, (space_size / n_points) ** (1.0 / ndim),
+                        dtype='float64')
```

Comment:
Is there a reason to specify float64 here and elsewhere?

Reply:
In `np.full(x, fill_value)` with the newest numpy versions the resulting dtype will be of dtype of `fill_value`. In this case I'm not sure it there is a combinations of parameters that could make it an int, but in any case it's safer to specify the dtype explicitly..

Reply:
Got it!